server/monitor_test.go

```diff
@@ -1313,18 +1313,18 @@ func TestConcurrentMonitoring(t *testing.T) {
 			for i := 0; i < 150; i++ {
 				resp, err := http.Get(url + endpoint)
 				if err != nil {
-					t.Fatalf("Expected no error: Got %v\n", err)
```

Comment:
@wallyqs  So do your want me to address this one in gnatsd or are you going to do it? Just checking.
PS: The bad part about this report is that - although correct - we will never see the benefit of the correction since if we hit that line (which would produce a race) it means that the test is failing and something needs to be fixed in the product anyway. I wish we could silence those specific reports ;-).